testing/internal/e2e/boundary/session_recording.go

```diff
@@ -0,0 +1,116 @@
+// Copyright (c) HashiCorp, Inc.
+// SPDX-License-Identifier: MPL-2.0
+
+package boundary
+
+import (
+	"context"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"testing"
+	"time"
+
+	"github.com/cenkalti/backoff/v4"
+	"github.com/hashicorp/boundary/api/sessionrecordings"
+	"github.com/hashicorp/boundary/testing/internal/e2e"
+	"github.com/stretchr/testify/require"
+)
+
+// GetSessionRecordingIdCli uses boundary CLI to list session recordings
+// and get the first in the specified organisation
+// Returns session recording ID.
+func GetSessionRecordingIdCli(t testing.TB, ctx context.Context, orgId string) string {
+	output := e2e.RunCommand(ctx, "boundary",
+		e2e.WithArgs(
+			"session-recordings", "list",
+			"-scope-id", orgId,
+			"-format", "json",
+		),
+	)
+	require.NoError(t, output.Err, string(output.Stderr))
+
+	var sessionRecordingListResult sessionrecordings.SessionRecordingListResult
+	err := json.Unmarshal(output.Stdout, &sessionRecordingListResult)
+	require.NoError(t, err)
+
+	return sessionRecordingListResult.Items[0].Id
```

Comment:
Maybe add a `require.GreaterOrEq(t, len(sessionRecordingListResult.Items), 1)` to prevent an unseemly panic here?